workshop/Lab0/README.md

```diff
@@ -0,0 +1,114 @@
+# Lab 0: Install Prequisites
+
+Before we begin, we need to install Service Catalog and a few other things
+we'll need. Towards that end, this lab contains instructions to install:
+
+* The Helm CLI, version 2.7.0 or later
+* The Helm tiller pod inside your cluster
+* Service Catalog, version 0.1.18 or later
+* svcat, the Service Catalog CLI tool
+* create a PersistantVolume on our cluster
+
+# Set KUBECONFIG
+
+Make sure your KUBECONFIG environment variable is set to the output of
+
+`bx cs cluster-config [cluster-name]`
+
+or otherwise targeting your IBM Cloud Kubernetes Service cluster.
+
+# Install Helm
+
+1. Download and install [the Helm CLI](https://github.com/kubernetes/helm#install).
+2. Run `helm init` to install the tiller pod into your kube cluster. This pod is Helm's entry point
+for creating additional infrastructure inside your cluster.
+3. To allow the tiller pod permissions to perform changes inside your cluster, run
```

Comment:
While I don't want to dive deep into RBAC concepts here, I think mentioning the term "RBAC" would be good so that they can tie it into the RBAC workshops/patterns when we create them.  So perhaps something like:
To allow the tiller pod permissions to perform changes inside your cluster, we need to define the Role Based Access Control (RBAC) policy for it by defining a new cluster role and giving it the appropriate permissions.